pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerImpl.java

```diff
@@ -920,6 +918,12 @@ public CompletableFuture<Void> connectionOpened(final ClientCnx cnx) {
         return future;
     }
 
+    protected CompletableFuture<ProducerResponse> askBrokerClearPreviousConsumer(ClientCnx cnx) {
+        long closeRequestId = client.newRequestId();
+        ByteBuf cmd = Commands.newCloseConsumer(consumerId, closeRequestId);
+        return cnx.sendRequestWithId(cmd, closeRequestId);
```

Comment:
This method just sends the `CLOSE_CONSUMER` command to broker.  The name `askBrokerClearPreviousConsumer` is not clear. It could be better to name it with `sendCloseConsumerCommand` and reuses it in `ConsumerImpl#closeAsync` as well.

Reply:
fixed